src/Agent.Plugins/PipelineArtifact/PipelineArtifactConstants.cs

```diff
@@ -11,5 +11,6 @@ public class PipelineArtifactConstants
         public const string RestoreCache = "RestoreCache";
         public const string RootId = "RootId";
         public const string SaveCache = "SaveCache";        
+        public const string FileShareArtifact = "FilePath";
```

Comment:
>FilePath [](start = 49, length = 8)

no artifact in the name?

Reply:
FileShare is on of the artifact type, therefore the naming.